images/indexer/Dockerfile

```diff
@@ -10,7 +10,7 @@ ENV HOME /opt/indexer
 WORKDIR /opt/indexer
 
 ENV DEBIAN_FRONTEND noninteractive
-RUN apk add --no-cache git bzip2 make bash libtool boost-dev autoconf automake g++
+RUN apk add --no-cache git make bash
```

Comment:
It looks like `libtool boost-dev autoconf automake g++` (everything except bzip2) is still needed to get sandbox running from source on my machine

Reply:
False alarm -- works when indexer is set to master, resolving